### ISSUE TITLE
User's Tumblr Import Site Displaying "Invalid Post Address" error

### ISSUE THREAD

**Issue by selectedselections** [Automattic/wp-calypso#28721]
#### Steps to reproduce
1.  User opens a post that contains images from their Tumblr import
2. When trying to modify their images or add new images, they immediately see an error message that says "Invalid Post Address: This post cannot be found. Check the web address or start a new post." 
3. They cannot edit the posts they have imported from Tumblr 

#### What I expected

User expected to be able to edit their imported post's media and content. They want to be able to edit posts they have imported from Tumblr.

#### What happened instead

After a few moments of working on a post, or as soon as they attempt to add media, they are shown an error message and can no longer edit: https://snag.gy/TpgUvV.jpg  

They are usually attempting to add/edit media  that made it over from the import when the error appears and they can no longer edit. It seems their media links are still referencing their Tumblr site which has since been deleted. The user no longer has access to the Tumblr export. 

#### Browser / OS version

User reports this issue in both Chrome and Firefox

#### Screenshot / Video

 https://snag.gy/TpgUvV.jpg

#### Context / Source

8131374-hc
1580361-zen

#user-report 

-->

**Comment by jenlynnemc** [comment]
Adding to Issue:

- Madeline used the steps described in the field guide to grab the user's Tumblr import files
- created a test import site for the user at: https://importandiamafraidofyou.wordpress.com
- sent the import files to the user and asked them to try and import again
- User experienced the same issue with the test import site

**Comment by xpurichan** [comment]
The user briefly returned to the ticket and reported that this has been resolved. However, they returned and report that it is still happening for some of their posts, see ticket for two provided examples.

In their example #1 I was initially not able to replicate the problem, I visited the editor for the page with no errors. I did see the error on example #2. Next I revisited the editor for example #1, and the error appeared.
